examples/pylint/E0704_misplaced_bare_raise.py

```diff
@@ -0,0 +1,12 @@
+class CustomException(Exception):
```

Comment:
This is interesting. We won't be strict on this in 148, so please disable the check.